style.css

```diff
@@ -0,0 +1,157 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&family=Pacifico&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Palette+Mosaic&display=swap');
+
+body{
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  min-height:100vh;
+  min-width:50vw;
+}
+
+main{
+  background-image: url(https://images.unsplash.com/photo-1619409437363-1b1d6e539836?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8MjN8fHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=500&q=60);
+  background-repeat: no-repeat;
+  background-size: 100% 88%;
+  font-family: 'Lato', sans-serif; 
+}
+
+nav{
+  position: fixed;
+  right: .1%;
+  background-color:rgba(65, 65, 65, 0.8);
+  border-radius: 1px;
+  color: rgb(228, 228, 236);
+  font-size: 1.5rem;
+  opacity: .6;
+  top: 0;
+  padding: 0px 5px 0px 5px;
+  } 
+
+  /* nav{
+    position: fixed;
+    right: .1%;
+    background-color:rgba(65, 65, 65, 0.8);
+    border-radius: 1px;
+    color: rgb(228, 228, 236);
+    font-size: 1.5rem;
+    opacity: .6;
+    top: 0;
+    padding: 0px 5px 0px 5px;
+    }  */
```

Comment:
Please Do not leave unused code as comments in your codebase. Unused code as comments can make your code look unclean.

Reply:
Your comments are polite and you use why :+1: